src/SoundComponent.cpp

```diff
@@ -1,10 +1,10 @@
 #include "SoundComponent.h"
+#include "ResourceManager.h"
 
-
-void SoundComponent::SetSound(const char* path, bool looping)
+void SoundComponent::SetSound(const char* name, bool looping)
 {
-	Buffer.loadFromFile(path);
-	Sound.setBuffer(Buffer);
+	Sound = ResourceManager::Get().GetSound(name);
```

Comment:
So we go back to the SoundBufffer, not Sound stuff :) Right now, ResourceManager holds one true Sound for everyone. So if there are 10 entities that want to use it, they all have the same settings for volume, attenuation, position, etc :) 

SoundComponent should have it's own sf::Sound, and instead it should grab the buffer
`Sound.setBuffer(ResourceManager::Get().GetSoundBuffer(name));`